packages/cosmic-swingset/README.md

```diff
@@ -102,11 +102,9 @@ Wait about 5 seconds for the chain to produce its first block, then switch to an
 make scenario2-run-client BASE_PORT=8000
 ```
 
-You can communicate with the node by opening http://localhost:8000/
-
-You can start other solo nodes with `make scenario2-run-client BASE_PORT=8001` and `make scenario2-run-client BASE_PORT=8002` and communicate with them respectively with on http://localhost:8001/ and http://localhost:8002/
-
+You can communicate with the node by opening the http://localhost:8000/ (`agoric open`)
 
+You can start other solo nodes with `make scenario2-run-client BASE_PORT=8001 SOLO_OTEL_EXPORTER_PROMETHEUS_PORT=9466` and `make scenario2-run-client BASE_PORT=8002 SOLO_OTEL_EXPORTER_PROMETHEUS_PORT=9467` and communicate with them respectively with on http://localhost:8001/ (`agoric open --hostport 127.0.0.1:8001`) and http://localhost:8002/ (`agoric open --hostport 127.0.0.1:8002`)
```

Comment:
Currently, the default port is fixed and when we run multiple instances (following this instruction) then that default port start to conflict with the first process. I now think the better fix will be to not have prometheus run by default so that we have not port conflict and simple commands to run here.

Reply:
Yes, if we can leave prometheus out altogether, that seems much better.